src/OpenSage.Game/Data/Map/NamedCamera.cs

```diff
@@ -9,28 +11,37 @@ public sealed class NamedCamera
         public Vector3 LookAtPoint { get; private set; }
         public string Name { get; private set; }
 
-        public Vector3 Unknown1 { get; private set; }
-        public Vector3 Unknown2 { get; private set; }
+        public Vector3 EulerAngles { get; private set; }
+        public float Zoom { get; private set; }
 
```

Comment:
Please could you add `Vector2 Unknown` here? The general pattern is to store the values, so they can be written back out. Otherwise the roundtrip tests will fail.